manifests/templates/otel-collector.yaml

```diff
@@ -93,7 +93,7 @@ spec:
     spec:
       containers:
       - name: otel-collector
-        image: gcr.io/config-management-release/otelcontribcol:v0.87.0-gke.2
```

Comment:
Where is the `-gke.N` suffix?

Reply:
+1. The image should be pushed by the Louhi flow, which appends `-gke.N`.

Reply:
Does it need it? This is just the regular tag. https://gke-internal.googlesource.com/config-sync/open-telemetry/opentelemetry-collector-contrib/+/refs/tags/v0.89.0

It wasn't clear to me if there all get built or if i need to make a release branch to trigger a gke build.

Reply:
hmm. maybe these don't exist

Reply:
It looks like that tag hasn't been imported yet. You don't need to make a release branch, it should be automated. It's possible something went wrong, I can take a closer look

Reply:
Either way, do we want to upgrade minor versions? I think we should be deliberate about upgrading minor versions with a review of potential breaking changes. If it's just for CVE fixes, it's lower risk to create a new patch. Not to say we shouldn't have a cadence for upgrading minor versions

Reply:
the branch didn't exist. so i just made one... should i not have?

Reply:
the louhi build isn't triggered yet. should i hit it manually or is it just delayed for polling?